core/actionresult/build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id "com.android.library"
+    alias libs.plugins.kotlinSerialization // todo can we centralize this?
```

Comment:
I tried adding `plugin(libs.plugins.kotlinSerialization.get().getPluginId())` in the json gradle, no luck

Reply:
Note: after rebuilding the project in Hedgehog I still had the lint warning. Upgrading to Iguana fixed it